dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -414,6 +414,16 @@ If you have providers as list of provider ids because you just released them you
 ```shell script
 cd "${AIRFLOW_REPO_ROOT}"
 
+breeze release-management publish-docs providers-index amazon cncf.kubernetes --override-versioned --run-in-parallel
+
+breeze release-management add-back-references providers-index amazon cncf.kubernetes
+```
+
+or with
+
+```shell script
+cd "${AIRFLOW_REPO_ROOT}"
+
```

Comment:
Whould we remove the "publish_provider_documentation.sh" and refereence to it now?

Reply:
for now lets leave it as an option.
I will do another cleanup on the whole doc soon